src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const stringModified = sourceString
+    .split(/;\s*/)
+    .filter((item) => item.trim() !== '');
```

Comment:
```suggestion
    .filter((item) => item.trim());
```
You don't need to compare it with an empty string
https://developer.mozilla.org/en-US/docs/Glossary/Truthy